superset/connectors/sqla/models.py

```diff
@@ -145,6 +145,27 @@ class TableColumn(Model, BaseColumn):
     update_from_object_fields = [s for s in export_fields if s not in ("table_id",)]
     export_parent = "table"
 
+    @property
+    def is_num(self) -> bool:
```

Comment:
nit: `is_numeric` seems better

Reply:
Good point; I thought these were part of the FAB model hence would be difficult to rename, but they were not.